core/src/stores/store.rs

```diff
@@ -111,6 +111,7 @@ pub trait Store {
         project: &Project,
         data_source_id: &str,
         limit_offset: Option<(usize, usize)>,
+        remove_system_tags: bool,
```

Comment:
Do we really need to go down to the `store` level? Seems easier to handle this entirely from `data_source.rs` ?

Reply:
Ah gotcha :+1: OK LGTM :+1: